org.okstar.ok-msg.yml

```diff
@@ -0,0 +1,112 @@
+id: org.okstar.ok-msg
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: ok-msg-desktop
+rename-icon: ok-msg
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=all
+  - --talk-name=org.gnome.Mutter.IdleMonitor
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.indicator.application
+  - --talk-name=org.ayatana.indicator.application
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=xdg-documents
+  - --filesystem=/usr/local
+  - --unset-env=QT_PLUGIN_PATH
+  - --env=PATH=/app/bin:/usr/bin
+
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: .
+    version: '23.08'
+    autodownload: true
+    autodelete: false
+
+build-options:
+  cflags: -DNDEBUG
+  cxxflags: -DNDEBUG
+  build-args:
```

Comment:
build-args doesn't belong here - I'm sure flatpak-builder will emit a warning at least.

Also not allowed on flathub.

Reply:
I ensure that `build-args` is normally in `build-options` section, that is a correct, and I do not know why that is not descripted in official flatpakbuilder document, otherwise the project can not be build because cmake need to fetch repository  webrtc and gloox from github.

Reply:
1. build-args doesn't belong to build-options
2. network it not available on flathub so:

> otherwise the project can not be build because cmake need to fetch repository webrtc and gloox from github.

this will not be possible either way. The build IS sandboxed and it's not an option.